neo_force_scheme/_neo_force_scheme.py

```diff
@@ -207,27 +208,39 @@ def transform(
             # randomly initialize the projection
             if starting_projection_mode == ProjectionMode.RANDOM:
                 # print(starting_projection_mode)
-                Xd = np.random.random((size, 2))
+                Xd = np.random.random((size, n_dimension))
             # initialize the projection with tsne
             elif starting_projection_mode == ProjectionMode.TSNE:
-                Xd = tsne.excute_tsne(X)
+                Xd = tsne.excute_tsne(X, n_dimension=n_dimension)
             # initialize the projection with pca
             elif starting_projection_mode == ProjectionMode.PCA:
-                Xd = pca.excute_pca(X)
+                Xd = pca.excute_pca(X, n_dimension=n_dimension)
         # create random index TODO: other than random
         index = np.random.permutation(size)
 
         if self.cuda:
             Xd, self.projection_error_ = self._gpu_transform(Xd, index=index, total=total, inplace=inpalce)
```

Comment:
Do you mind adding a "raise NotImplementedException()" here if n_dimension != 2 and cuda==True?